src/ChannelServer/World/Inventory.cs

```diff
@@ -209,19 +220,62 @@ public Item GetItem(EquipSlot slot)
 		/// </summary>
 		/// <param name="item"></param>
 		/// <return>Index of the item.</return>
-		public int AddSilent(Item item)
+		public Dictionary<int, int> AddSilent(Item item)
 		{
 			var cat = item.Data.Category;
-
 			lock (_syncLock)
 			{
+				Dictionary<int, int> itemsUpdated = new Dictionary<int, int>();
```

Comment:
just stupid rules xD

Firstly i write 
Dictionary<int, int> itemsUpdated;

but then VS scream on me that it want initializtion.